<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxConsum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Consum</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDetritus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Detritus</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDOC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DOC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHerbivore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Herbivore</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMacro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWaterdecomp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Waterdecomp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSeddecomp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Seddecomp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSedconsumer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sedconsumer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPhyto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Phyto</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPOC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>POC</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTSS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>TSS:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTSS">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMacroTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro Temperature:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMacroTemp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGrossMacroCoef">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelGrossMacroCoef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Gross Macro Coef:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRespMacroCoef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Respiration Macro Coef:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSenescenceMacroCoef">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Senescence Macro Coef:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMacroMassMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro Mass Max:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMacroVelocityMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro Velocity Max:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelKPhyto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Phyto:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelKMacro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>K-Macro:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRespMacroCoef">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSenescenceMacroCoef">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMacroMassMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKPhyto">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMacroVelocityMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKMacro">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectHydroMap">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Hydro Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelectedMap">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Map:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHydroMap">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDaysToRun">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Days to run:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDaysToRun">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddHydroMap">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>110</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Hydro Map to Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemoveHydroMap">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>300</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Selected Hydro Map</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAdjacentCells">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjacent Cells Only?</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTimestep">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFreq">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>140</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Frequency (# Days):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputFreq">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimestep">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestep:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectTempFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Temperature File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectPARFile">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select PAR File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelectedTemp">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Temperature:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTempFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>120</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSelectedPAR">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected PAR:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPARFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserErrors">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>230</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;None&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrors">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Errors:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetHydroMap">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>140</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Configuration"/>
    <addaction name="actionLoad_Configuration"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSelectHydroMap</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectHydroMapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddHydroMap</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addHydroMapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRemoveHydroMap</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeHydroMapClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectTempFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectTemperatureFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectPARFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectPARFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRun</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>runClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTimestep</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>timestepUpdate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectHydroMapClicked()</slot>
  <slot>addHydroMapClicked()</slot>
  <slot>removeHydroMapClicked()</slot>
  <slot>selectTemperatureFileClicked()</slot>
  <slot>selectPARFileClicked()</slot>
  <slot>runClicked()</slot>
  <slot>timestepUpdate(int)</slot>
 </slots>
</ui>
